### DISCUSSION TITLE
After upgrading to v1.20.0, the node have not mined one block yet, 3 days.

### DISCUSSION THREAD

**Discussion by maweis1981** [filecoin-project/lotus#10447] [Storage Provider]
After I upgrade to v1.20.0 from 3 Mar , the node have not mined one block yet, 

Miner ID: f0859013

`root@LSMINER197:~/backup# lotus-miner -v
lotus-miner version 1.20.0+mainnet+git.33bec58be`

the Latest Log:

`{"level":"info","ts":"2023-03-06T21:28:42.775+0800","logger":"miner","caller":"miner/miner.go:475","msg":"completed mineOne","tookMilliseconds":2604,"forRound":2660098,"baseEpoch":2660097,"baseDeltaSeconds":10,"nullRounds":0,"lateStart":false,"beaconEpoch":2755943,"lookbackEpochs":900,"networkPowerAtLookback":"22011988999658242048","minerPowerAtLookback":"3377184324452352","isEligible":true,"isWinner":true,"error":null}
{"level":"info","ts":"2023-03-06T23:51:41.818+0800","logger":"miner","caller":"miner/miner.go:475","msg":"completed mineOne","tookMilliseconds":1813,"forRound":2660384,"baseEpoch":2660383,"baseDeltaSeconds":10,"nullRounds":0,"lateStart":false,"beaconEpoch":2756229,"lookbackEpochs":900,"networkPowerAtLookback":"22031607178039787520","minerPowerAtLookback":"3377184324452352","isEligible":true,"isWinner":true,"error":null}
{"level":"info","ts":"2023-03-07T17:19:50.654+0800","logger":"miner","caller":"miner/miner.go:475","msg":"completed mineOne","tookMilliseconds":10647,"forRound":2662480,"baseEpoch":2662479,"baseDeltaSeconds":10,"nullRounds":0,"lateStart":false,"beaconEpoch":2758325,"lookbackEpochs":900,"networkPowerAtLookback":"22036811874376548352","minerPowerAtLookback":"3377184324452352","isEligible":true,"isWinner":true,"error":null}
{"level":"info","ts":"2023-03-08T08:31:41.606+0800","logger":"miner","caller":"miner/miner.go:475","msg":"completed mineOne","tookMilliseconds":1601,"forRound":2664304,"baseEpoch":2664303,"baseDeltaSeconds":10,"nullRounds":0,"lateStart":false,"beaconEpoch":2760149,"lookbackEpochs":900,"networkPowerAtLookback":"22041327687196180480","minerPowerAtLookback":"3377184324452352","isEligible":true,"isWinner":true,"error":null}
fil@LSMINER197:~$ sudo -s`

**Comment by stuberman** [comment]
I see you have 3 PiB QAP (also raw)

I upgraded today to Lotus v1.20.03 and won a block

Check your time on your machine is it in sync with an atomic clock?

I see you were a winner - look at your logs: 
{"level":"info","ts":"2023-03-08T08:31:41.606+0800","logger":"miner","caller":"miner/miner.go:475","msg":"completed mineOne","tookMilliseconds":1601,"forRound":2664304,"baseEpoch":2664303,"baseDeltaSeconds":10,"nullRounds":0,"lateStart":false,"beaconEpoch":2760149,"lookbackEpochs":900,"networkPowerAtLookback":"22041327687196180480","minerPowerAtLookback":"3377184324452352","isEligible":true,**"isWinner":true**,"error":null}

**Reply by maweis1981** [reply]
That is the last winner block before v1.20.0. Bro

**Reply by TippyFlitsUK** [reply]
Stuarts question is still very valid. How are you syncing your server time and are you in sync?

**Reply by Angelo-gh3990** [reply]
You have mined 9 blocks in 7 days and 2 today according to : https://filfox.info/en/address/f0859013

**Reply by lbj2004032** [reply]
> You have mined 9 blocks in 7 days and 2 today according to : https://filfox.info/en/address/f0859013

Have you encountered this problem

After the upgrade, start the lotus-miner log card in the

2023-03-16T22:24:35.165+0800 INFO stores paths/index.go:181 New sector storage: 12db6be1-710e-4dbb-9be6-3618dcd916cd

**Reply by lbj2004032** [reply]
> Have you encountered this problem
> 
> After the upgrade, start the lotus-miner log card in the
> 
> 2023-03-16T22:24:35.165+0800 INFO stores paths/index.go:181 New sector storage: 12db6be1-710e-4dbb-9be6-3618dcd916cd

Have you encountered this problem

After the upgrade, start the lotus-miner log card in the

2023-03-16T22:24:35.165+0800 INFO stores paths/index.go:181 New sector storage: 12db6be1-710e-4dbb-9be6-3618dcd916cd
